clang/test/3C/base_subdir/canwrite_constraints.c

```diff
@@ -35,8 +35,8 @@ int *bar(int *q) {
 
 int gar(intptr a) {
         int* b = a;
-        //CHECK_LOEWR int* b = a;
-        //CHCEK_HIGHER _Ptr<int> b = a;
+        //CHECK_LOWER int* b = a;
+        //CHECK_HIGHER _Ptr<int> b = a;
```

Comment:
I'm unsure whether FileCheck recognizes these comments as they are written now, but let's be consistent anyway.
```suggestion
        // CHECK_LOWER: int* b = a;
        // CHECK_HIGHER: _Ptr<int> b = a;
```